.github/workflows/release-srtool.yml

```diff
@@ -76,9 +78,20 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Prepare build options
+        run: |
+          BUILD_OPTIONS=""
+          if [[ ${{ matrix.chain }} == 'asset-hub-rococo' || ${{ matrix.chain }} == 'asset-hub-westend' || ${{ matrix.chain }} == 'rococo' || ${{ matrix.chain }} == 'westend' ]]; then
+            BUILD_OPTIONS="${{ inputs.build_opts }}"
```

Comment:
Conceptually it seems a bit odd that it ignores the build opts for some runtimes. Maybe you can add a `METADATA_HASH_ENABLED` flag or something?

Reply:
We should add `on-chain-release-build` to every runtime and use this one here.

Reply:
This was the initial plan, the reason why I went this way was, that it includes this feature `sp-api/disable-logging` that turns off the logging, that is on the other hand needed for devops guys cc: @PierreBesson. After discussion with Pierre, that was kinda compromised solution🙈 to not build two types of runtimes (with on chain feature and with logging). If there is another to do it, I'll be happy to use it:). Otherwise, I can also add additional step to build those runtimes with logs for devops.

Reply:
We are also running logging nodes for main net, because we have logging disabled there. However, we can also just remove the "disable-logging" feature there.

Reply:
I've tested the solution with two separate jobs, building runtimes with and without logging (having `on-chain-release-build` option). It looks fine. The only thing now is that `people-rococo` and `poepele-westend` seems not have this feature. Whom could I contact to ask to add this option? Is it Joe?

Reply:
You can just copy the feature declaration from the other runtimes.

Reply:
And then we should remove the `disable-logging` everywhere.

Reply:
Basti, I've added the `on-chain-release-build` where it was missing, should the `disable-logging` deletion be done here as well or as a separate task? I'm not sure if it is worth to mention it in the rel notes or not.